server/Mistakes.Journal.Api/Api/Mistakes/Mappers/WeatherMapper.cs

```diff
@@ -0,0 +1,67 @@
+﻿using System;
+using System.Linq;
+using Mistakes.Journal.Api.Api.Mistakes.WebModels;
+using Newtonsoft.Json.Linq;
+
+namespace Mistakes.Journal.Api.Api.Mistakes.Mappers
+{
+    public static class WeatherMapper
+    {
+        // https://openweathermap.org/weather-conditions
+        private static readonly long[] GoodWeatherCodes = { 600, 800, 801 };
+
+        public static WeatherWebModel ToWeatherWebModel(string openWeatherJson, string sunsetJson)
```

Comment:
The point of those methods it to be extension methods. Now you want to create static class with static method. I think, that you could create two different extension methods on string. one for openWeatherJson and one for sunsetJson. WeatherWebmodel would be named differently (it is not only weather, maybe UserLocationDataWebModel, because it is based on some location he provides). It would have two internal classes TimeOfDayWebmodel and WeatherWebmodel that are returned from the extensions I described earlier.

Reply:
I'm not sure it is worth. Currently, all "dynamic" things are aggregated and I think it ok